### DISCUSSION TITLE
UAE Booster Data

### DISCUSSION THREAD

**Discussion by jarnold231** [owid/covid-19-data#2223] [Q&A] [answered]
Hi there, 

Interested to know where the data for UAE booster doses is being sourced from. The link in the locations.csv file doesn't appear to show information on boosters and I've been unable to find a public source for the information so far.

Thanks
J

**Comment by edomt** [comment, marked as answer]
Hi @jarnold231 

The number of boosters is inferred from the public data at https://fcsc.gov.ae/en-us/Pages/Covid19/UAE-Covid-19-Updates.aspx
Since the UAE is only using 2-dose vaccines, if you have the total number of doses administered, the number of first doses administered (our `people_vaccinated`), and the number of second doses administered (our `people_fully_vaccinated`), then you can calculate `total_boosters = total_vaccinations - people_vaccinated - people_fully_vaccinated`

**Reply by jarnold231** [reply]
Thank you!
